### PR TITLE
update load data local infile docs

### PR DESCRIPTION
**Pull request by lacrimosaprinz**
## What type of PR is this?

- [ ] Enhancement
- [ ] Displaying
- [ ] Typo
- [x] Doc Request

## Which issue(s) this PR fixes:

issue #

### CODE REVIEW

File: docs/MatrixOne/Develop/import-data/bulk-load/0.7-load-jsonline.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+|String| TEXT (without knowing the limit of this string)|
+|String| DATETIME or TIMESTAMP (with format as "YYYY-MM-DD HH:MM:SS.XXXXXX")|
+|String| DATE (with format as "YYYY-MM-DD")|
+|String| TIME (with format as "HH-MM-SS.XXXXXX")|
+|Number| INT (with interger numbers)|
+|Number| FLOAT or DOUBLE (with floating numbers) |
+|Boolean| BOOL(true/false)|
+|Object| Not supported|
+|Array| Not supported|
+|Null| Not supported|
```
Comment (mooleetzi) [inline docs/MatrixOne/Develop/import-data/bulk-load/0.7-load-jsonline.md:100]: Object, Array & Null are all supported, `Object` and `Array` ->  must be json type, Null is ok for all type.

File: docs/MatrixOne/Reference/SQL-Reference/Data-Manipulation-Language/0.7-load-data.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+        [TERMINATED BY 'string']
+        [[OPTIONALLY] ENCLOSED BY 'char']
+        [ESCAPED BY 'char']
+    ]
+    [LINES
+        [STARTING BY 'string']
+        [TERMINATED BY 'string']
+    ]
+    [IGNORE number {LINES | ROWS}]
+    [PARALLEL {'TRUE' | 'FALSE'}]
```
Comment (mooleetzi) [inline docs/MatrixOne/Reference/SQL-Reference/Data-Manipulation-Language/0.7-load-data.md:23]: load data local 的parallel暂时还不支持

File: docs/MatrixOne/Develop/import-data/bulk-load/0.7-load-jsonline.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    true	1	var	2020-09-07	2020-09-07 00:00:00	2020-09-07 00:00:00	18	121.11
+    true	1	var	2020-09-07	2020-09-07 00:00:00	2020-09-07 00:00:00	18	121.11
+    ```
+
+!!! note
+    If you use Docker to launch MatrixOne, when you try to import the jsonline file, please make sure that you have a data directory mounted to the container. You can check on the [load csv tutorial](load-csv.md) about the loading with docker installation.
+
+## Constraints
+
+1. `Load data` doesn't support JSON type in schema.
```
Comment (mooleetzi) [inline docs/MatrixOne/Develop/import-data/bulk-load/0.7-load-jsonline.md:200]: load data csv支持json类型，但是需要确保json内不含有字段终止符号
csv
```
{"a":1, "b":2}, 2
```
createsql
`create table t1(a json, b int);`

load sql
`load data infile '$path' into table t1` 会失败，因为json内存在默认终止符`,`

如果将csv中的json用双引号引起来
```
"{"a":1, "b":2}", 2
```
使用上面的load sql就可以导入成功；或者将csv中的字段终止符改为json中不存在的符号，然后使用`fields terminated by`在load sql中手动指定终止符

Comment (lacrimosaprinz) [reply docs/MatrixOne/Develop/import-data/bulk-load/0.7-load-jsonline.md]: i add content in [0.7-load-csv.md] as below:

## Constraints

The loaded csv file format supports JSON. However, you need to ensure that the JSON does not contain field termination symbols. If the JSON does contain field termination symbols, wrap the JSON with double quotation marks. For example:

- Right csv file example: `"{"a":1, "b":2}", 2`
- Wrong csv file example: `{"a":1, "b":2}, 2`

**Review by dengn — APPROVED** [review]

**Pull request merged**